### DISCUSSION TITLE
没法查看MDK的Armclang和armcc链接后输出的map信息。xmake -v -D -w 没有链接器的回显输出

### DISCUSSION THREAD

**Discussion by PR2403** [xmake-io/xmake#4410] [Q&A] [answered]
这是xmake屏蔽输出了吗，可以的话希望有个参数能够让xmake完全回显输出

**Comment by PR2403** [comment]
我手动执行了xmake最后的链接指令，是有输出的
![image](https://github.com/xmake-io/xmake/assets/64511917/6efb9fa9-36b9-4d53-a523-41bed5327c5b)
但是xmake没有
![image](https://github.com/xmake-io/xmake/assets/64511917/37023a47-033f-4eb2-a1cb-42f2d951c369)

**Reply by waruqi** [reply, marked as answer]
目前不支持输出，你可以提 feature request 到 issues
